src/mpyl/cli/build.py

```diff
@@ -123,11 +123,28 @@ def shell_complete(self, ctx: click.Context, param, incomplete: str):
     type=Pipeline(),
     required=False
 )
+@click.option(
+    '--test', '-t',
+    help='A specific test version on https://test.pypi.org/project/mpyl/ to use for the build.',
+    type=click.STRING,
+    required=False
+)
+@click.option(
+    '--arguments', '-a',
+    multiple=True,
+    help='A series of arguments to pass to the pipeline. Note that will run within the pipenv in jenkins. '
+         'To execute `mpyl build status`, pass `-a run -a mpyl -a build -a status`',
+)
```

Comment:
Couldn't this be just a string?

Reply:
No, unfortunately not. That's not how `click` works apparently.